.github/workflows/maven.yml

```diff
@@ -19,9 +19,15 @@ jobs:
       matrix:
         # test against latest update of each major Java version, as well as specific updates of LTS versions:
         os: [ubuntu-18.04, windows-latest]
-        java: [ 8, 11 ]
-        RUNTIME: [ ol, wlp ]
-        RUNTIME_VERSION: [ 20.0.0.6, 20.0.0.9 ]
+        java: [8, 11]
```

Comment:
Not sure which is more desirable, but I think if `java` is moved below `RUNTIME_VERSION`, we might be able to get a better at-a-glance look at the builds in the github actions UI.

Currently the liberty version is cut off. 
![image](https://user-images.githubusercontent.com/689163/101063699-b7160580-3558-11eb-85e0-fb2a352b7fbf.png)